plugins/BEdita/Core/src/Model/Behavior/UniqueNameBehavior.php

```diff
@@ -54,14 +55,13 @@ class UniqueNameBehavior extends Behavior
      */
     public function uniqueName(EntityInterface $entity)
     {
-        $config = $this->getConfig();
         $uname = $entity->get('uname');
         if (empty($uname)) {
-            $uname = $this->generateUniqueName($entity, $config, false);
+            $uname = $this->generateUniqueName($entity);
         }
-        $id = !(empty($entity->get('id'))) ? $entity->get('id') : null;
+        $id = !empty($entity->get('id')) ? $entity->get('id') : null;
```

Comment:
`$entity->get('id')` behaves exactly like this (returns `null` if `id` is not set) and it doesn't add complexity, nor triggers warning because `empty()` wants variables to be passed by reference.